app/src/main/java/com/example/georgesamuel/dubaihotels/presentation/features/details/HotelDetailsActivity.java

```diff
@@ -0,0 +1,115 @@
+package com.example.georgesamuel.dubaihotels.presentation.features.details;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.LinearLayoutCompat;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.georgesamuel.dubaihotels.presentation.features.map.MapActivity;
+import com.example.georgesamuel.dubaihotels.R;
+import com.example.georgesamuel.dubaihotels.entities.Hotel;
+import com.squareup.picasso.Picasso;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+import static com.example.georgesamuel.dubaihotels.presentation.features.map.MapActivity.HOTEL_NAME_CONSTANT;
+import static com.example.georgesamuel.dubaihotels.presentation.features.map.MapActivity.LATITUDE_CONSTANT;
+import static com.example.georgesamuel.dubaihotels.presentation.features.map.MapActivity.LONGITUDE_CONSTANT;
+
+public class HotelDetailsActivity extends AppCompatActivity  {
+
+    @BindView(R.id.hotel_name_details_text_view)
+    TextView hotelNameTextView;
+
+    @BindView(R.id.actual_price_text_view)
+    TextView actualPriceTextView;
+
+    @BindView(R.id.hotel_address_text_view)
+    TextView addressTextView;
+
+    @BindView(R.id.normal_price_text_view)
+    TextView normalPriceTextView;
+
+    @BindView(R.id.details_pic_image_view)
+    ImageView hotelPicImageView;
+
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+
+    @BindView(R.id.back_image_view)
+    ImageView backImageView;
+
+    @BindView(R.id.open_map_button)
+    Button openMapButton;
+
+    boolean isImageFitToScreen;
+    private Hotel hotel;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_details);
+        ButterKnife.bind(this);
+        if (getIntent() != null) {
+             hotel = (Hotel) getIntent().getSerializableExtra("Details");
```

Comment:
use EXTRA_HOTEL_DETAILS as static constant

Reply:
ok , i updated it